src/main/java/uk/gov/justice/hmpps/prison/health/CacheInfoContributor.java

```diff
@@ -1,48 +1,15 @@
 package uk.gov.justice.hmpps.prison.health;
 
-import net.sf.ehcache.CacheManager;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.actuate.info.Info;
 import org.springframework.boot.actuate.info.InfoContributor;
 import org.springframework.stereotype.Component;
 
 import java.lang.management.ManagementFactory;
-import java.util.HashMap;
-import java.util.Map;
 
 @Component
 public class CacheInfoContributor implements InfoContributor {
-
-    private final CacheManager cacheManager;
-
-    @Autowired
-    public CacheInfoContributor(final CacheManager cacheManager) {
-        this.cacheManager = cacheManager;
-    }
-
     @Override
     public void contribute(final Info.Builder builder) {
-        final Map<String, String> results = new HashMap<>();
```

Comment:
not sure about this at present - have removed it for now. I think the information can be obtained from the metrics in app insights which would be a better place for it. Will have to come back to it later...